### PR TITLE
Create cloud foundry

### PR DESCRIPTION
**Pull request by nonococoleo**
# Pull Request Template

In this PR, the project can running as a ibm cloud foundry app, and elephantSQL is used as database service. IBM cloud support added in VM.

## Checklist:

- [x] I have written test cases for this change, and my code passed the test
- [x] I have updated documentation to reflect this change
- [x] I have attached this PR to an issue/user story

### CODE REVIEW

**Review by KHafner — APPROVED** [review]

File: runtime.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+python-3.7.9
```
Comment (iLtc) [inline runtime.txt:1]: This does not match our development version. 

Please change this to `3.6.9` or change the python version in both `vagrant` and `travis ci` to `3.7.9`

**Review by iLtc — CHANGES_REQUESTED** [review]

File: runtime.txt (hunk 1 above)
Comment (nonococoleo) [reply runtime.txt]: use 3.6.9 for the project

**Review by iLtc — APPROVED** [review]
LGTM, thanks!

**Review by jules203 — APPROVED** [review]
Great work!

**Comment by KHafner** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/b5c2987e-b3d1-4601-9f28-34419832137d?env_id=ibm:yp:us-south "nyu-shopcart-service-f20-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/c869c127-ceda-423d-80b8-f74ee52b7378 "nyu-shopcart-service-f20-pipeline") deployed [shopcarts](https://github.com/NYU-DevOps2020-shopcarts/shopcarts/releases/tag/deploy-ibm-yp-us-south-kfh226-nyu.edu-dev-20201103-180801 "deploy-ibm-yp-us-south-kfh226-nyu.edu-dev-20201103-180801") to [dev](https://console.bluemix.net/apps/9acc4a4a-3e86-4e84-9d4f-87cc363a812b?env_id=ibm:yp:us-south "ibm:yp:us-south:kfh226@nyu.edu:dev"), including commit b56ca67583d564bc1874d20ca7ae55753b2a04f3

**Comment by KHafner** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/b5c2987e-b3d1-4601-9f28-34419832137d?env_id=ibm:yp:us-south "nyu-shopcart-service-f20-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/c869c127-ceda-423d-80b8-f74ee52b7378 "nyu-shopcart-service-f20-pipeline") deployed [shopcarts](https://github.com/NYU-DevOps2020-shopcarts/shopcarts/releases/tag/deploy-ibm-yp-us-south-kfh226-nyu.edu-prod-20201208-144028 "deploy-ibm-yp-us-south-kfh226-nyu.edu-prod-20201208-144028") to [prod](https://console.bluemix.net/apps/ac06a6b3-040d-43d7-b290-6bfbba3f8c41?env_id=ibm:yp:us-south "ibm:yp:us-south:kfh226@nyu.edu:prod"), including commit b56ca67583d564bc1874d20ca7ae55753b2a04f3

**Pull request merged**